admin_guide/managing_projects.adoc

```diff
@@ -94,3 +101,55 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+== Node Selectors
```

Comment:
Same here, wrt gerund/anchor:

```
[[using-node-selectors-with-projects]]
== Using Node Selectors with Projects
```

Or possibly simply "Using Node Selectors"... A reason to also include the "with Projects", even if it seems redundant in a topic called "Managing Projects", is that the concept of "node selector" appears in other, non-strictly-project-related material.

Or maybe, "Setting Node Selectors for Projects".